package.json

```diff
@@ -1093,19 +1142,7 @@
           "type": "boolean",
           "scope": "resource",
           "default": false,
-          "description": "%Automatically open Workunits on creation%",
-          "deprecationMessage": "%Use WUSubmitAction instead%"
-        },
-        "ecl.WUShowResults": {
-          "type": "string",
-          "scope": "resource",
-          "enum": [
-            "internal",
-            "external",
-            "disabled"
-          ],
-          "default": "internal",
-          "description": "%Show results after submitting a workunit%"
+          "description": "%Automatically open Workunits on creation%"
```

Comment:
Should be "Automatically open Results on creation"?

Reply:
Agreed. That would be clearer although that was not part of this task.